modular_citadel/code/modules/client/loadout/_service.dm

```diff
@@ -4,6 +4,12 @@
 	path = /obj/item/clothing/under/color/grey
 	restricted_roles = list("Assistant")
 
+/datum/gear/staffassistanttide
```

Comment:
Hey, could you just change the above gear datum's path with this one instead? Don't worry, it was just a stupid meme I made months ago, you have my blessing to remove it.